app/controllers/albums.js

```diff
@@ -44,3 +44,27 @@ exports.buy = (request, response) => {
     })
     .catch(error => handleError(error, 'Unexpected error when fetching'));
 };
+
+exports.boughts = (request, response) => {
+  if (!request.userLogged)
+    return response
+      .status(400)
+      .send(errors.authenticationError(['You must be logged to see the purchased albums']));
+  if (!request.userLogged.isAdmin && Number(request.params.user_id) !== request.userLogged.id)
+    return response
+      .status(400)
+      .send(errors.listAlbumError('You are not allowed to see albums purchased from other users'));
+  Album.findByUserId(request.params.user_id)
+    .then(purchasedAlbums => {
+      if (purchasedAlbums.length === 0)
+        return response.status(200).send('The user has not bought any album yet');
+      albumService
+        .getAll(response)
+        .then(albums => albums.filter(album => purchasedAlbums.find(pAlbum => pAlbum.idAlbum === album.id)))
```

Comment:
I think the card is not very clear about this, but you are supposed to return the bought albums that are stored locally in the database after buying them, not their counterparts in the album service